src/app/index.html

```diff
@@ -3,10 +3,10 @@
   overlayOpacity="0.4"
   backgroundImageUrl="~/assets/hero-sample-1.jpg">
   <stache-hero-heading>
-    Stache
+      Stache<sup><small>2</small></sup>
```

Comment:
Not loving the exponent 2,  let's just leave it as Stache 2 for right now (regular 2).